src/components/ReactTags.js

```diff
@@ -354,21 +414,24 @@ class ReactTags extends Component {
     const { classNames } = this.state;
     const moveTag = allowDragDrop ? this.moveTag : null;
     return tags.map((tag, index) => {
-      return (
-        <Tag
-          key={`${tag.id}-${index}`}
-          index={index}
-          tag={tag}
-          labelField={labelField}
-          onDelete={this.handleDelete.bind(this, index)}
-          moveTag={moveTag}
-          removeComponent={removeComponent}
-          onTagClicked={this.handleTagClick.bind(this, index)}
-          readOnly={readOnly}
-          classNames={classNames}
-          allowDragDrop={allowDragDrop}
-        />
-      );
+      return this.state.editIndex !== index ?
+        (
+          <Tag
+            key={`${tag.id}-${index}`}
+            index={index}
+            tag={tag}
+            editTag={this.editTag.bind(this, index)}
+            labelField={labelField}
+            enableEdit={this.props.enableEdit ? this.props.enableEdit : false}
```

Comment:
can you set the value of editable to be  false in default props instead of deciding what to pass here.